CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.6.2 (2020-02-10)
+
+- Fixed the inline delete component to not watch for focus events until is has been fully displayed. [#73](https://github.com/blackbaud/skyux-layout/pull/73)
```

Comment:
Fixed the inline delete component to watch for focus events after they fully display.

Reply:
Done